README.md

```diff
@@ -5,7 +5,7 @@ Content management is possible using hidden `'/editor'` route. To use this route
 ```
 JWT_HMAC_KEY= // !!!SECRET BACKEND ONLY!!! should be unique and secure, used to sign JWT. The HMAC is used for symmetric encryption of tokens, if it is leaked, your app is insecure.
 
-REACT_APP_VALID_HOSTED_DOMAIN= // email address domain allowed for admins
+REACT_APP_VALID_HOSTED_DOMAIN= // email address domain allowed for admins	REACT_APP_VALID_HOSTED_DOMAIN= // email address domain allowed for admins
```

Comment:
This line is still duped.  Perhaps you forgot to push some local changes?
(Generally the requester resolves comments, so as to prevent this kind of issue.)